robosats/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: ${APP_ROBOSATS_IP}
+      APP_PORT: ${APP_ROBOSATS_PORT}
+      ROBOSATS_ONION: robosats6tkf3eva7x2voqso3a5wcorsnw34jveyxfqi2fu7oyheasid.onion
```

Comment:
I think this should be under the `web` service's environment.

Reply:
Indeed. Commited a fix, thanks for reviewing!